java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2164_java.html

```diff
@@ -16,6 +16,12 @@
 
 double d = (double)a + (double)b;
 </pre>
+<h2>Exceptions</h2>
```

Comment:
This is not yet implemented, and part of [SONARJAVA-2323](https://jira.sonarsource.com/browse/SONARJAVA-2323).